src/UI/Components/PaginationWidget/Provider.tsx

```diff
@@ -1,7 +1,8 @@
 import React from "react";
+import { Pagination as PaginationComponent } from "@patternfly/react-core";
```

Comment:
Was the name reassignment really needed?

Reply:
We have interface with the same name, so I decided that is better to change this component name than hunt for interface across the app :D

Reply:
Fair enough :D